external/slowparse/slowparse.js

```diff
@@ -1301,7 +1301,17 @@
             "section", "table", "ul"],
       "th": ["th", "td"],
       "td": ["th", "td"],
-      "li": ["li"]
+      "tr": ["tr"],
+      "li": ["li"],
+      "rb": ["rb","rt","rtc","rp"],
+      "rt": ["rb","rt","rtc","rp"],
+      "rtc": ["rb","rtc","rp"],
+      "rp": ["rb","rt","rtc","rp"],
+      "optgroup": ["optgroup"],
+      "option": ["option"],
+      "thead": ["tbody", "tfoot"],
+      "tbody": ["tbody", "tfoot"],
+      "tfoot": ["tbody"]
```

Comment:
Shouldn't "tr", "rb", "rt", "rtc", "rp", "optgroup", "thead", "tbody" and "tfoot" also be added to `ombittableCloeTagHtmlElements`?